File: Olympiad/POI/official/2013-XX/gdz/dlazaw/gdz.cpp
// gdz.cpp - Przykladowe użycie biblioteki do zadania GDZ
#include "cgdzlib.h"

int main(void) {
    int n = inicjuj(), mini = 1, i;
    for (i = 2; i <= n; ++i) {
        if (g(mini, i)) {
            mini = i;
        }
    }
    odpowiedz(mini);
    return 0;
}

File: Olympiad/POI/official/2013/gdz/dlazaw/gdz.pas
{ gdz.pas - Przykladowe uzycie biblioteki do zadania GDZ }
program gdz;

uses pgdzlib;

var i, n, mini : LongInt;

begin
    n := inicjuj();
    mini := 1;
    for i := 2 to n do
        if g(mini, i) = 1 then
            mini := i;
    odpowiedz(mini);
end.

